workshops/models.py

```diff
@@ -317,6 +317,13 @@ class Event(models.Model):
     latitude = models.FloatField(null=True, blank=True)
     longitude = models.FloatField(null=True, blank=True)
 
+    REPO_REGEX = re.compile(r'https://github\.com/(?P<name>[^/]+)/'
```

Comment:
Use `https?` in case they give us an `http://` URL instead of an `https://` URL.